### ISSUE TITLE
How to Log: commands inside observations are never executed, and travel onward with a "learning"'s authority

### ISSUE THREAD

**Issue by thobi1987-prog** [rebelytics/one-skill-to-rule-them-all#166]
**Target:** `SKILL.md` → `## How to Log` (file format / mandatory fields)
**Verified against:** origin/main, re-checked 14 Sep 2026 — `How to Log` still carries
no rule about commands inside observations and no `commands_verified:` field (mandatory
fields there: id, title, status, type, skill, proposes_skill, siblings_checked, date,
parked_until). The Pre-Flight Principle in `references/skill-authoring.md` covers
commands embedded in *skills*, at authoring time.
**Adjacent, not duplicate:** #149 (backticks in observation bodies execute when the
file is written through an unquoted heredoc — same surface, different failure), and #107 (instrument-failure guards enumerated per snippet)
— that is about guards inside snippets; this is about snippets that never ran at all.

## What happened

Three observations were turned into a new skill. The authoring pre-flight
("execute every embedded command once against real data") caught **three wrong
commands in a row**, and all three came out of the observation bodies:

1. A validator invoked with a flag that does not exist on the distribution in
   question — the documented dry-run flag there is a different letter. The
   observation had confidently named the wrong one.
2. A backgrounded process started through a wrapper, then addressed via `$!` —
   which refers to the wrapper, not the process. The proof-of-life check
   reports "not running" and the cleanup finds nothing, while everything is
   fine.
3. A cache-reading command used as an inventory, which returns a small fraction
   of the population unless a prior step populates the cache, and an
   unfiltered majority of empty entries if one filter is missing.

All three read as correct. All three would have shipped had the target skill
not happened to be authored in a session that could run them.

## Why the current text did not prevent it

`How to Log` enforces two checks at write time — the target skill must exist
(`skill:`), and siblings must be considered (`siblings_checked:`, mandatory and
never blank, precisely so that "checked" and "never considered" stop being
byte-identical). It enforces nothing about **commands**.

The Pre-Flight Principle does cover commands, but it fires when a *skill* is
written. Between logging and authoring, the wrong command sits in the log — and
it arrives there with the authority of a first-hand observation. What was
actually witnessed was the *experience*; the command was reconstructed from
memory, often after the fact. Nothing downstream distinguishes the two.

The asymmetry that makes this worth a rule: at **logging** time the session is
standing in front of the very system the command refers to, and running it
costs seconds. At **authoring** time that system is frequently unreachable — a
different machine, a different day, a different person. The check is being
deferred from the one moment it is nearly free to a moment where it is
expensive or impossible.

## Suggested fix

In `How to Log`, alongside the existing write-time checks: if an observation
contains an executable command, it must have been **run once, literally, with
its output looked at**, before the file is written. Where that is not possible,
the command is marked as unverified in the body, so the later pre-flight knows
where to look.

A `commands_verified:` frontmatter field would give this the same enforcement
shape as `siblings_checked:` — including the value that makes the *absence* of
the judgement visible:

```yaml
commands_verified: "crontab -n: run, exit 0/1 as documented | setsid+$!: NOT RUN, no host available"
```

and `none` where the observation carries no commands. Same reasoning as the
sibling field: a body with an unrun command is byte-identical whether it was
checked and trusted or never considered.

## Principle

An intermediate store that later produces instructions transmits not only
content but **authority**: what arrives as an observation from a real session
is treated downstream as established, when what was established was the
experience, never the command. Verification belongs where the evidence still
exists — at writing time, in front of the system — not where the knowledge is
consumed, where the system is often out of reach. Deferred, it is performed at
best by someone without access, at worst by nobody.

**Comment by rebelytics** [comment]
Thanks — the commands-in-observations gap is a real one. In the queue for one of the next weekly releases.
